kotlin-eclipse-ui/src/org/jetbrains/kotlin/utils/MessageDialogContentUtils.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.kotlin.utils
+
+import org.eclipse.core.runtime.CoreException
+import org.eclipse.core.runtime.MultiStatus
+
+object MessageDialogContentUtils {
+
+    @JvmStatic
+    fun getErrorTitleForMessageDialog(error: Throwable): String = error.localizedMessage
+
+    @JvmStatic
+    fun getErrorDescriptionForMessageDialog(error: Throwable): String = when(error) {
+        is CoreException -> when (error.status) {
+            is MultiStatus -> join(error.status.children.filterNot { it.isOK }.map { it.message }, "\n")
+            else -> error.status.message
+        }
+        else -> error.localizedMessage
+    }
```

Comment:
It's a bit not "Kotlin-way" :)
First, I'm not sure that `object` is needed here, top-level functions/properties are fine.
Next, `getError...` looks like a usual Java-getter, in Kotlin you can use properties for this. Plus, this is a matter of taste, but I would do so:
```
val Throwable.errorTitleForMessageDialog: String 
    get() = localizedMessage

val Throwable.errorDescriptionForMessageDialog: String
    get() = ... 
```

Reply:
You're right, making extensions out of this does look nicer.